pkgs/tools/misc/env-lscolors/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  pname = "env-lscolors-unstable";
+  version = "2022-05-05"; # upstream don't do releases
```

Comment:
```suggestion
  pname = "env-lscolors";
  version = "unstable-2022-05-05"; # upstream don't do releases
```

Reply:
Fine. By the way, we have 95 more packages that do it other way round:
```
; git grep -- '-unstable'|grep pname|wc -l
95
; git grep 'unstable-'|grep version|wc -l
959
```